include/nitro/lang/string.hpp

```diff
@@ -102,5 +102,19 @@ namespace lang
         return result;
     }
 
+    inline bool starts_with(const std::string& full, const std::string& beginning)
+    {
+        return full.find(beginning) == 0;
+    }
+
+    inline void replace_all(std::string& str, const std::string& from, const std::string& to)
```

Comment:
Please find better names for `from` and `to`

Reply:
Isnt the "Egg" doesnt start with "Egg and Spam" an example for a longer search than base string?